### DISCUSSION TITLE
[835520]  Dodge Show

### DISCUSSION THREAD

**Discussion by zzzgg** [pjy612/SteamManifestCache#586] [Share]
### Steam 游戏ID

`appid:835520`

### 密钥列表

```hash
835521;3fdd981cd05b208700be5f5f40c77df32a7b1d254c8f69c6857f4774cfe48f44
```

### 清单附件

[835521_6342104533155815783.zip](https://github.com/pjy612/SteamManifestCache/files/14965104/835521_6342104533155815783.zip)

### 备注

练练走位的小游戏
emai：zaxsqd123@gmail.com
CODE：7C981E2C9CAB60EB

**Comment by pjy612** [comment]
* 密钥 分号 调整一下 #26

**Reply by zzzgg** [reply]
改好了

**Reply by zzzgg** [reply]
可以麻烦拉一下进内测吗

**Comment by zzzgg** [comment]
能拉进内测吗，我发了邮箱但你没回



---Original---
From: ***@***.***&gt;
Date: Sat, Apr 13, 2024 13:18 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [pjy612/SteamManifestCache] [835520]  Dodge Show (Discussion#586)




 
密钥 分号 调整一下

**Reply by pjy612** [reply]
带有目的性的分享 可以帮忙删帖的呢 亲，需要删帖吗？
